packages/datatrak-web/src/components/CodeGeneratorQuestion/CodeGeneratorQuestion.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import { TextField, Typography } from '@material-ui/core';
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import { SHORT_ID, generateMongoId, generateShortId } from './generateId';
+import { SurveyQuestionInputProps } from '../../types';
+
+const LabelId = styled(Typography).attrs({
+  variant: 'h1',
+  color: 'textPrimary',
```

Comment:
In theory you shouldn't need to add `color` to this, since it uses the `primary` color by default